src/Okta.DNX.OAuth.ResourceServer/Controllers/TodoController.cs

```diff
@@ -42,6 +43,7 @@ public IActionResult GetById(string id)
             return new ObjectResult(item);
         }
 
+        [Authorize("todo.write")]
```

Comment:
We should call out these extra permissions (write and delete) in the configuration part of the readme as well